.agent-os/instructions/core/plan-product.md

```
@@ -0,0 +1,407 @@
+### Step 4: Create tech-stack.md
+
+Use the file-creator subagent to create the file: .agent-os/product/tech-stack.md and use the following template:
+
+<file_template>
+  <header>
+    # Technical Stack
+  </header>
+</file_template>
+
+<required_items>
+  - application_framework: string + version
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)